README.md

```diff
@@ -9,7 +15,8 @@ Use the following pom snippet to include Ahab in your project:
 <dependency>
     <groupId>org.renci.ahab</groupId>
     <artifactId>ahab</artifactId>
-    <version>0.1.2</version>
+    <version>0.1.6</version>
```

Comment:
@hinchliff - Other code references `<version>0.1.7-SNAPSHOT</version>`, should this be consistent? or nah?

Reply:
well, 0.1.6 is the current release...  and it doesn't make sense to have the README reference a snapshot version.
but the README lagging the latest release is a problem that i don't really know a good (simple) solution too.